src/prescriptions/prescription-tab-lists.component.tsx

```diff
@@ -39,12 +47,12 @@ const PrescriptionTabLists: React.FC = () => {
             {tabs.map((tab, index) => {
               return (
                 <Tab
-                  title={t(tab.label)}
+                  title={t(tab.key)}
                   key={index}
```

Comment:
Would it make sense to use the key and the key here, now that we have one?

Reply:
Yes, this makes sense